feature-detects/indexeddb.js

```diff
@@ -16,7 +16,12 @@ define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
   // - Firefox shipped moz_indexedDB before FF4b9, but since then has been mozIndexedDB
   // For speed, we don't test the legacy (and beta-only) indexedDB
 
-  var indexeddb = prefixed('indexedDB', window);
+  var indexeddb;
+  try {
+    indexeddb = prefixed('indexedDB', window);
+  } catch (e) {
+  }
```

Comment:
we could return, but we aren't inside of an addTest, so I don't think `return false` would be doing what you think it would be doing